tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -402,3 +402,83 @@ def _verify_built_artifact(self, build_dir, function_logical_id, expected_files)
         all_artifacts = set(os.listdir(str(resource_artifact_dir)))
         actual_files = all_artifacts.intersection(expected_files)
         self.assertEquals(actual_files, expected_files)
+
+
+class TestBuildCommand_SingleFunctionBuilds(BuildIntegBase):
+    template = "many-functions-template.yaml"
+
+    EXPECTED_FILES_GLOBAL_MANIFEST = set()
+    EXPECTED_FILES_PROJECT_MANIFEST = {'__init__.py', 'main.py', 'numpy',
+                                       # 'cryptography',
```

Comment:
why is cryptography commented out?

Reply:
This was copied from the above python tests.

This is the specific reason: https://github.com/awslabs/aws-sam-cli/blob/develop/tests/integration/testdata/buildcmd/Python/requirements.txt#L4